Web-Files/6/Dickinson6Experiment4ebb.xsl

```diff
@@ -0,0 +1,354 @@
+                                            </xsl:when>
+                                            <xsl:when test=".[contains(., 'CPa.')]">
+                                                <div class="modalDesc">Complete Poems 'A'</div>
+                                            </xsl:when>
+                                            <xsl:when test=".[contains(., 'CPb.')]">
+                                                <div class="modalDesc">Complete Poems 'B'</div>
+                                            </xsl:when>
+                                        </xsl:choose>
+                                    </div>
+                                </xsl:for-each>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+            </body>
+        </html>
+    </xsl:template>
+
+    <xsl:template match="q[@rend]">
+        <xsl:choose>
+            <xsl:when test="@rend = 'single'">
+                <q class="single">
+                    <xsl:apply-templates/>
+                </q>
+            </xsl:when>
+            <xsl:otherwise>
+                <q class="double">
+                    <xsl:apply-templates/>
+                </q>
+            </xsl:otherwise>
+        </xsl:choose>
+    </xsl:template>
+
+
+    <xsl:template match="head">
+        <h2>
+            <span id="poem{//idno}">
+                <xsl:apply-templates select="title"/>
+            </span>
+        </h2>
+    </xsl:template>
+
+    <xsl:template match="lg">
+        <div class="para">
+            <xsl:apply-templates/>
+        </div>
+    </xsl:template>
+
+    <xsl:template match="l">
+        <div class="line">
+            <span class="lineNum"><xsl:value-of select="count(preceding::l) + 1"/><xsl:text>: </xsl:text></span>
+            <div class="lineContent">
+        <xsl:variable name="current" select="current()"/>
+                <!--<xsl:sequence select="current()"/>
+        </xsl:variable> -->
+       <!--2016-04-20 ebb: It doesn't seem to be necessary to declare this variable a sequence because of course current() is going to be a new
+           value each time the template match on `<l>` fires.-->
+         
+                <xsl:for-each select="$witness"><table border="1">
+                    <!-- ebb: removed @class="current()" from table element to output variants only inside cells.-->
+            <tr>
+                   <!--<xsl:value-of select="current()"/>-->                 
+                    <xsl:apply-templates select="$current" mode="row">
+                        <xsl:with-param name="wit" select="current()" as="xs:string" tunnel="yes"/>
+                    </xsl:apply-templates>
+                </tr></table>
+            </xsl:for-each>
+            </div>
+        </div>
+    </xsl:template>
+    <xsl:template match="l" mode="row">
+        <xsl:param name="wit" tunnel="yes"/>
```

Comment:
@nlottig94 @HelenaSabel So **here's** where we invoke the xsl parameter value through the tunnel!